test/modules/attributes/AttributesController.test.ts

```diff
@@ -650,6 +556,636 @@ export class AttributesControllerTest extends IntegrationTest {
+                                value: {
+                                    "@type": "Nationality",
+                                    value: "DE"
+                                },
+                                owner: CoreAddress.from("address")
+                            })
+                        })
+                        const successorData: IAttributeSuccessorParams = {
+                            content: IdentityAttribute.from({
+                                value: {
+                                    "@type": "Nationality",
+                                    value: "DE"
+                                },
+                                owner: CoreAddress.from("differentAddress")
+                            })
+                        }
+
+                        const validationResult =
+                            await consumptionController.attributes.validateAttributeSuccessionCommon(
+                                predecessor.id,
+                                successorData
+                            )
+                        expect(validationResult.isError()).to.be.true
+                        expect((validationResult as ErrorValidationResult).error.code).equals(
+                            "error.consumption.attributes.successionMustNotChangeOwner"
+                        )
+                    })
+                    it("should catch attempted change of content type", async function () {
+                        const predecessor = await consumptionController.attributes.createLocalAttribute({
+                            content: IdentityAttribute.from({
+                                value: {
+                                    "@type": "Citizenship",
+                                    value: "DE"
+                                },
+                                owner: CoreAddress.from("address")
+                            })
+                        })
+                        const successorData: IAttributeSuccessorParams = {
+                            content: RelationshipAttribute.from({
+                                key: "DisplayName",
+                                value: {
+                                    "@type": "ProprietaryString",
+                                    value: "ADisplayName",
+                                    title: "Display Name"
+                                },
+                                owner: CoreAddress.from("address"),
+                                confidentiality: RelationshipAttributeConfidentiality.Public
+                            })
+                        }
+
+                        const validationResult =
+                            await consumptionController.attributes.validateAttributeSuccessionCommon(
+                                predecessor.id,
+                                successorData
+                            )
+                        expect(validationResult.isError()).to.be.true
```

Comment:
we have a custom matcher for validation-results and you should use it everywhere

=> 
```suggestion
                        expect(validationResult).to.be.an.errorValidationResult()
```

the cool thing is that you can also directly check for the error code and message using this matcher. So you should move the next lines content to this.

Same counts for success, there is a matcher `.to.be.a.successfulValidationResult` please use this everywhere too.